test/routes/connecte/routesConnecteApiTeleversement.spec.js

```diff
@@ -17,17 +17,18 @@ describe('Les routes connecté de téléversement', () => {
       testeur.depotDonnees().nouveauTeleversementServices = async () => {};
     });
 
-    describe('Quand requête POST sur `/api/televersement/services`', () => {
-      it("vérifie que l'utilisateur est authentifié", (done) => {
-        testeur.middleware().verifieRequeteExigeAcceptationCGU(
-          {
-            method: 'post',
-            url: 'http://localhost:1234/api/televersement/services',
-          },
-          done
-        );
-      });
+    // Comme le middleware est placé au niveau de l'instanciation du routeur, un seul test suffit
+    it("vérifie que l'utilisateur est authentifié", (done) => {
```

Comment:
`it("vérifie que l'utilisateur est authentifié, 1 seul test suffit car le middleware est placé au niveau de l'instanciation du routeur", (done) => {`